dist/redhat/build_rpm.sh

```diff
@@ -68,7 +68,7 @@ if [[ ! -f /usr/bin/pystache ]]; then
     pkg_install python2-pystache || pkg_install pystache
 fi
 
-echo "Ruuning unit tests"
+echo "Runing unit tests"
```

Comment:
example of fixing typo and introducing new one :)))
Run**n**ing

Reply:
LOL